saml/models/core/response_test.go

```diff
@@ -71,14 +71,6 @@ var responseXMLStatus = `<?xml version="1.0" encoding="UTF-8"?>
    </saml2p:Status>
 </saml2p:Response>`
 
-func Test_ParseResponse_Status(t *testing.T) {
```

Comment:
there doesn't appear to be a `Status` field.

Reply:
I wonder why. It was def. defined once. Will add it back, but thx for fixing this. I can add tests back once we have the status field added again.